src/layouts/IndexLayout.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import styled from "styled-components";
+import Link from "next/link";
+import TEMP_FEATURED_DATA from "./../../projects.json";
+import ProjectPreview from "./../components/project/ProjectPreview";
+import Button from "../components/general/Button.js";
+
+function renderProjects() {
```

Comment:
Could this instead be a component that returns a fragment?

```jsx
function ProjectPreviewList() {
  return (
    <>
      {TEMP_FEATURED_DATA.projects.map(project => (
        <ProjectPreview ... />
      )}
    </>
  );
}
```

Reply:
That seems like a bit of overkill, I generally like to keep render functions like this in the same component. Especially when this will most likely be the only one and it's fairly simple.